android/build.gradle.kts

```
@@ -73,8 +73,8 @@ android {
         }
     }
     compileOptions {
-        sourceCompatibility = JavaVersion.VERSION_1_8
-        targetCompatibility = JavaVersion.VERSION_1_8
+        sourceCompatibility = JavaVersion.VERSION_17
+        targetCompatibility = JavaVersion.VERSION_17
     }
     kotlinOptions {
         jvmTarget = Versions.JVM_TARGET
```

we can set it directly here

```suggestion
        jvmTarget = JavaVersion.VERSION_17.toString()
```